app/src/constants/i18n.ts

```diff
@@ -48,3 +48,10 @@ export const EditLocationBoundaryI18N = {
   createErrorText:
     'An error has occurred while attempting to edit your location boundary, please try again. If the error persists, please contact your system administrator.'
 };
+
+export const EditSpeciesI18N = {
+  editTitle: 'Edit focal and ancillary species',
```

Comment:
I think we should be consistent with casing here, so either:
1) `Edit Species`
or
2) `Edit Focal and Ancillary Species`

Reply:
Probably `Edit Species` as they are bundled together under a single 'Species` section